iconrpcserver/utils/icon_service/converter.py

```diff
@@ -138,6 +120,19 @@ def _convert_value_integer(value):
         return int(value, 16)
 
 
+def _convert_value_integer_str(value):
+    if isinstance(value, int):
+        return str(value)
+    if isinstance(value, str):
+        try:
+            if not value.startswith('0x') and not value.startswith('-0x'):
```

Comment:
```python
if value.startswith("0x") or value.startswith("-0x"):
    return int(value, 16)
else:
    return int(value)
```

Reply:
Resolved. I've changed them to str additionally.